terraform/modules/happy-service-eks/variables.tf

```diff
@@ -198,6 +198,23 @@ variable "routing" {
   })
   description = "Routing configuration for the ingress"
 }
+
+variable "sidecars" {
+  type = map(object({
+    image : string
```

Comment:
Sidecars are generally images that are hosted somewhere else right? They aren't images built as part of the happy project?

Reply:
Sidecars are not built as a part of the happy pipeline. Typically, an agent of some sort.